test/TesterInternal/General/RequestContextTest.cs

```diff
@@ -169,22 +181,33 @@ public void Dispose()
 
             IRequestContextTestGrain grain = this.fixture.GrainFactory.GetGrain<IRequestContextTestGrain>(GetRandomGrainId());
 
+#if NETSTANDARD
+            RequestContext.ActivityId.Value = activityId;
+#else
             Trace.CorrelationManager.ActivityId = activityId;
+#endif
            Assert.Null(RequestContext.Get(RequestContext.E2_E_TRACING_ACTIVITY_ID_HEADER));
             Guid result = await grain.E2EActivityId();
             Assert.Equal(activityId,  result);  // "E2E ActivityId not propagated correctly"
             RequestContext.Clear();
 
+#if NETSTANDARD
+            RequestContext.ActivityId.Value = nullActivityId;
```

Comment:
Yes I can do a wrapper. 
As for using AsyncLocal, I think the back-compat issue here is how to set ActivityId from 1.5 on.
I think we are not questioning the reliability of using AsyncLocal to store ActivityId, right? 
I would vote to keep ActivityId as part of RequestContext, which means to set ActivityId like:
`RequestContext.ActivityId.Value = nullActivityId`,
instead of using 
`Trace.CorrelationManager.ActivityId = nullActivityId`

it seems to me is a better design, even though it is a breaking change.

Or we can do `RequestContext.ActivityId = nullActivityId`, by modifying our current implementation of `RequestContext.ActivityId`